cartoframes/data/clients/bigquery_client.py

```diff
@@ -93,7 +93,14 @@ def download_to_file(self, job, file_path, fail_if_exists=False, column_names=No
             rows = self._download_by_bq_storage_api(job)
         except Exception:
             log.debug('Cannot download using BigQuery Storage API, fallback to standard')
-            rows = job.result()
+
+            try:
+                rows = job.result()
+            except Exception:
+                if job.errors:
+                    log.error([error['message'] for error in job.errors if error['message']])
+
+                raise CartoException('Error downloading data from BigQuery')
```

Comment:
Do we want to tell the user about BigQuery?

Reply:
umm, it is not adding value to the client and we can avoid it. I am going to change it